app/components/fullNode/FullNodeJointStyles.js

```diff
@@ -0,0 +1,204 @@
+// @flow
+import styled, { css } from 'styled-components';
+import { smColors } from '/vars';
+
+const Actionable = css`
+  &:hover {
+    opacity: 0.8;
+  }
+  &:active {
+    opacity: 0.6;
+  }
+`;
+
+export const PageWrapper = styled.div`
+  display: flex;
+  flex-direction: row;
+  justify-content: space-around;
+  height: 100%;
+`;
+
+export const LeftPane = styled.div`
+  text-align: left;
+  flex: 2;
+  padding: 24px 0;
+  margin-right: 32px;
+`;
+
+export const RightPane = styled.div`
+  flex: 1;
+  background-color: white;
+  padding: 30px;
+  min-height: 480px;
+  max-height: 75%;
+  border: 1px solid ${smColors.borderGray};
+`;
+
+export const RightPaneInner = styled.div`
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+  height: 100%;
+`;
+
+export const LeftPaneInner = styled.div`
+  display: flex;
+  flex-direction: column;
+  justify-content: flex-start;
+  height: 100%;
+`;
+
+export const RightHeaderWrapper = styled.div`
+  padding: 0 12px 12px 12px;
+`;
+
+export const LeftHeaderWrapper = styled.div`
+  padding: 12px 0;
+  margin-top: 12px;
+`;
+
+export const RightHeaderText = styled.span`
+  font-size: 14px;
+  font-weight: bold;
+`;
+
+export const BaseText = styled.span`
+  font-size: 16px;
+  font-weight: normal;
+`;
+
+export const GrayText = styled(BaseText)`
+  color: ${smColors.textGray};
+`;
+
+export const BoldText = styled.span`
+  font-size: 16px;
+  font-weight: bold;
+`;
+
+export const SemiBoldText = styled.span`
+  font-size: 16px;
+  font-weight: 600;
+`;
+
+export const ActionLink = styled(BaseText)`
+  user-select: none;
+  color: ${smColors.green};
+  cursor: pointer;
+  ${Actionable}
+`;
+
+// $FlowStyledIssue
+export const BaseImage = styled.img`
+  width: ${({ width }) => width || 42}px;
+  height: ${({ height }) => height || 42}px;
+`;
+
+export const ImageWrapper = styled.div`
+  display: flex;
+  flex-direction: row;
```

Comment:
this prop is unnecessary as per component usage
add align-items: center; and width: 100%; to make inner image to be centered automaticly